### ISSUE TITLE
Its really anoying , idk whats the problem but the cache seem to be recreated when remove or add mods into a large modpack

### ISSUE THREAD

**Issue by quentin452** [kuba6000/Mobs-Info#48]
it is possible to add dynamic caching to reduce loading time : see 47

**Comment by kuba6000** [comment]
Cache is recreated when you add/remove or update any mod in the mod list by default. This behaviour can be changed in the config file `mobsinfo.cfg`.
There is currently no option to only update cache with mobs added by a new mod, maybe thats a good idea to do, I will think about it.

**Comment by quentin452** [comment]
# When will cache regeneration trigger? 0 - Never, 1 - ModAdditionRemoval, 2 - ModAdditionRemovalChange, 3 - Always, 
    I:CacheRegenerationTrigger=2
    so you mean i need to change the value 2 to 3?

**Comment by kuba6000** [comment]
No, if you don't want to regenerate a cache when a mod is added or deleted then change it to 0. Then the cache will only be regenerated when you remove the cache file from the config folder.
